src/conversions/jiff.rs

```diff
@@ -0,0 +1,1348 @@
+#![cfg(feature = "jiff-01")]
+
+//! Conversions to and from [jiff](https://docs.rs/jiff/)’s `Span`, `SignedDuration`, `TimeZone`,
+//! `Offset`, `Date`, `Time`, `DateTime`, `Zoned`, and `Timestamp`.
+//!
+//! # Setup
+//!
+//! To use this feature, add this to your **`Cargo.toml`**:
+//!
+//! ```toml
+//! [dependencies]
+//! jiff = "0.1"
+#![doc = concat!("pyo3 = { version = \"", env!("CARGO_PKG_VERSION"),  "\", features = [\"jiff-01\"] }")]
+//! ```
+//!
+//! Note that you must use compatible versions of jiff and PyO3.
+//! The required jiff version may vary based on the version of PyO3. Jiff also requires a MSRV
+//! of 1.70.
+//!
+//! # Example: Convert a `datetime.datetime` to jiff `Zoned`
+//!
+//! ```rust
+//! # use jiff_01 as jiff;
```

Comment:
You might need to cfg this doctest as not windows, IIRC windows needs a `tzdata` package but it's quite a faff to bother requiring it.